### DISCUSSION TITLE
Fixed symbolic link uncertainty for LFS

### DISCUSSION THREAD

**Discussion by seanking2919** [git-lfs/git-lfs#5093] [General]
For some unknown reason, we have this issue with what's supposed to be a symbolic link on the Pleroma website.

This file `apple-touch-icon.png` is meant to be a symbolic link to `pleroma_logo_vector_bg_180.png`. But instead, it is this:

`apple-touch-icon.png -> 'version https://git-lfs.github.com/spec/v1'$'\n''oid sha256:4eed2b6a032a028ae4f4d3c0d9408a13d59d7de5183ff1d1e17bcdeb8a2cf335'$'\n''size 30'$'\n'`

So, I did the following process:
- `rm -f apple-touch-icon.png`
- `ln -s pleroma_logo_vector_bg_180.png apple-touch-icon.png`
- Staged, committed, and pushed per usual

However, whether it's part of LFS or not does not seem clear. One on hand, it shows this in the "Changes" tab of my MR:
![image](https://user-images.githubusercontent.com/31253278/185288294-5cea664e-ed4c-4728-98fc-a76087aad906.png)

Then on the other, it shows the new file isn't part of LFS if I go to the MR branch:
![image](https://user-images.githubusercontent.com/31253278/185288425-c4dadd82-132c-449b-9806-ebeae06db81c.png)

Should I be worried about this or will it be fine with GitLab's LFS once it merges?
https://git.pleroma.social/pleroma/pleroma.social/-/merge_requests/69

**Comment by chrisd8088** [comment]
You might want to check with GitLab's support team on this question, I think.
